libmcount/pyhook.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Python binding for mcount entry and exit
+ *
+ * Copyright (C) 2017, LG Electronics, Honggyu Kim <hong.gyu.kim@lge.com>
+ *
+ * Released under the GPL v2.
+ */
+#include <stdio.h>
+#include <stdlib.h>
+#include <assert.h>
+#include <stdbool.h>
+#include <dlfcn.h>
+
+#include "pyhook.h"
+#include "utils/utils.h"
+
+/* python file name */
+const char *pyfile;
+
+static bool python_enabled = false;
+
+static void *python_handle;
+static const char *libpython = "/usr/lib/python2.7/config-x86_64-linux-gnu/libpython2.7.so";
```

Comment:
Is this really needed?  I think dynamic linker should know the directory (or some env variable?).  And I prefer supporting python 3 if possible..